<?xml version="1.0" encoding="UTF-8"?>

<mule xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
	xmlns:munit="http://www.mulesoft.org/schema/mule/munit" xmlns:munit-tools="http://www.mulesoft.org/schema/mule/munit-tools"
	xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
	xsi:schemaLocation="
		http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
		http://www.mulesoft.org/schema/mule/munit http://www.mulesoft.org/schema/mule/munit/current/mule-munit.xsd
		http://www.mulesoft.org/schema/mule/munit-tools  http://www.mulesoft.org/schema/mule/munit-tools/current/mule-munit-tools.xsd">
	<munit:config name="global-config-test-suite.xml" />
	<munit:test name="global-error-suiteTest-APIKIT:BAD_REQUEST" doc:id="d1dd66c1-7405-4850-b4d7-9b64e0ce844b" expectedErrorType="APIKIT:BAD_REQUEST">
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="25a0aa03-61ec-4d51-8de0-bfa3036c940f" processor="apikit:router">
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="APIKIT:BAD_REQUEST" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="60186fc1-8050-4c90-8e28-26848a71bf4d" processor="flow-ref">
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="0449cfa8-82d9-4d8d-88ac-30752eea9f03" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="89bd411b-01a4-47cf-8cb6-b64a2ba200e9" name="doit-hubspot-sapi-main"/>
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-APIKIT:METHOD_NOT_ALLOWED" doc:id="a128cd12-aba0-417e-b19b-95cdc22b1bcd" expectedErrorType="APIKIT:METHOD_NOT_ALLOWED" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="f8654f09-8d59-4762-9a28-96c495ed8c2e" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="APIKIT:METHOD_NOT_ALLOWED" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="76b3db71-3d50-4bce-b375-b90b6ec4d211" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="b24310dc-3151-4a8f-9f96-473e902df402" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="8a69dd7b-2bd6-424f-a45c-12541564e389" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-APIKIT:NOT_ACCEPTABLE" doc:id="05390c5d-b517-4707-9697-ca431e61f7ea" expectedErrorType="APIKIT:NOT_ACCEPTABLE" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="7f13bd9e-3834-4b82-be9b-5eeda709e3c2" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="APIKIT:NOT_ACCEPTABLE" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="62be6701-eb1c-4b52-ac20-0fafe90f5ebd" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="188ce12e-40ae-45a3-ad89-42e8287178c5" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="cb47c844-ec77-4999-bb0a-ea7aea348b06" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-APIKIT:NOT_FOUND" doc:id="b705cd21-d5a6-4b01-87f1-af5f78cfafb0" expectedErrorType="APIKIT:NOT_FOUND" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="34b99e81-7549-4e3f-bea0-5ae038a34c0b" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="APIKIT:NOT_FOUND" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="6d0e62f5-9eae-4e90-95be-fa9fdc235d87" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="f93c47be-0ce6-42b4-8b37-c4d9c8776322" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="6d35b2f9-78b8-45e8-80fd-4075f325177e" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-APIKIT:UNSUPPORTED_MEDIA_TYPE" doc:id="4cd22080-7585-4896-8622-7243bda65d35" expectedErrorType="APIKIT:UNSUPPORTED_MEDIA_TYPE" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="1dcf720e-4db6-4568-b684-6e2974137053" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="APIKIT:UNSUPPORTED_MEDIA_TYPE" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="e5f1d9a2-0b32-4e5d-98cb-8789f3dd1d15" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="22a9eec7-e5c9-4864-a290-f45ff1432b5d" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="0718b995-b1aa-4a23-8208-b832e11af594" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-APIKIT:NOT_IMPLEMENTED" doc:id="5787488e-6ab3-4f7c-9234-5364305a6bb2" expectedErrorType="APIKIT:NOT_IMPLEMENTED" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="06fa2c66-f854-4b7d-82ac-f646eeb4e499" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="APIKIT:NOT_IMPLEMENTED" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="b7d56122-eca9-44f2-b728-9e4785da6e35" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="df58cc7e-c2d5-493e-b707-92cd5ed81605" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="18ffc47e-c935-4ba2-ae72-4fb8964fc2e2" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
<!-- [STUDIO:"global-error-suiteTest-ANY"]	<munit:test name="global-error-suiteTest-ANY" doc:id="ae6bd6ef-4bf9-459f-ba51-d6bee69cb94d" expectedErrorType="ANY" >
		<munit:behavior >
			<munit:set-event doc:name="Set Event" doc:id="d30d8bdb-e39c-4ee6-93b4-bcc5937dcd22" >
				<munit:variables >
					<munit:variable key="webhookPayload" value="#[output application/json &#45;&#45;- readUrl('classpath://startcallflowtest\mock_variable_1.dwl')]" encoding="UTF-8" mediaType="application/json" />
				</munit:variables>
			</munit:set-event>
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="f42bec30-7e27-4491-9229-e070a7461300" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="rge-external-synch-eapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="ANY" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="6f8981d8-a055-4595-900a-e7d78961536f" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="f27add74-c59f-44b6-a8e4-218167172ccc" attributeName="doc:id" />
				</munit-tools:with-attributes>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock request-to-slack" doc:id="7cf3bd45-6d1e-4e9f-9311-5fdccf470cbd" processor="http:request">
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="3a4232fd-3709-4b5f-81d4-d6dd65a7bd05" attributeName="doc:id" />
				</munit-tools:with-attributes>
			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="d4a7532f-eb67-46ee-a747-00acd92b78d6" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test> [STUDIO] -->
	<munit:test name="global-error-suiteTest-HTTP:BAD_REQUEST" doc:id="75bec5fa-23f8-450b-a9a1-c81ab106bf43" expectedErrorType="HTTP:BAD_REQUEST" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="d7e19465-7df0-485f-b0f7-6d9e36393ad6" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="HTTP:BAD_REQUEST" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="12438d9c-ad50-4a0e-b60a-fba6d9d95958" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="ae6e972a-cbea-4a09-8c66-9a91384967c8" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="ff125c6f-3950-4b1c-983d-d7c013745288" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-HTTP:CONNECTIVITY" doc:id="b2dc25b8-944f-4594-9bf5-b698816b3aac" expectedErrorType="HTTP:CONNECTIVITY" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="1640481b-d1b3-4d16-a4a7-43717ac9c49c" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="HTTP:CONNECTIVITY" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="9b5b9903-4bc9-4a38-bd37-10204a453afd" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="ee5a1f4f-620d-4c2f-b2a8-3288ed134da7" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="46c0cf93-2804-4763-a83e-468bec1ee361" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-HTTP:FORBIDDEN" doc:id="907c8bbc-99e2-43ba-b348-70f943222b20" expectedErrorType="HTTP:FORBIDDEN" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="92576987-33dc-4308-9244-5c882ff71905" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="HTTP:FORBIDDEN" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="9f027a89-0732-4215-9741-418e7098c7b4" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="fb2d9406-6b74-4fdf-bee3-7710d0beaa11" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="6920149f-ca22-4b4b-9442-d89bb5880e37" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-HTTP:INTERNAL_SERVER_ERROR" doc:id="80965eb3-65be-49bb-90a7-4bb0ead0f6c5" expectedErrorType="HTTP:INTERNAL_SERVER_ERROR" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="375256a8-c336-4074-970f-b4db5fdcd873" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="HTTP:INTERNAL_SERVER_ERROR" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="f76a62cf-c337-4ace-a598-99beb206b1d4" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="fb2d9406-6b74-4fdf-bee3-7710d0beaa11" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="0aab9ee8-e475-4b26-a51a-6ce4c712ab42" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-HTTP:METHOD_NOT_ALLOWED" doc:id="201b1d71-d4c7-4aad-8988-8161162f0232" expectedErrorType="HTTP:METHOD_NOT_ALLOWED" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="f3ad88d1-aab6-4f19-8aa1-a792f1a3d95d" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="HTTP:METHOD_NOT_ALLOWED" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="fb91568c-c094-44d9-a4b8-3c7f8daa16ac" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="97ac28b9-73da-453d-a8e3-8213d4051a6a" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="7d483957-abfd-43e2-8110-2c409d15b050" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-HTTP:NOT_ACCEPTABLE" doc:id="17ab6eb9-7317-4da1-9ae6-01fe04f40fdd" expectedErrorType="HTTP:NOT_ACCEPTABLE" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="513587ff-f5ac-427e-b3a3-514cd449059f" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="HTTP:NOT_ACCEPTABLE" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="0a854d50-f08b-4242-aa84-e66340facb05" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="d42739c6-422e-4127-9dbf-43622b0f92b5" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="ccceed58-c9ea-41d9-b795-9d3ea7adc012" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-HTTP:RETRY_EXHAUSTED" doc:id="b5026bac-18b8-4da1-abcc-e380fc36d161" expectedErrorType="HTTP:RETRY_EXHAUSTED">
		<munit:behavior>
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="e0366e14-df87-4d10-90f3-f441ea86210d" processor="apikit:router">
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="HTTP:RETRY_EXHAUSTED" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="88e4c48e-0c1d-4096-9be8-5a84be6bd32f" processor="flow-ref">
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="29ca979c-8b5e-4618-9cab-13c9fe8a94fa" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution>
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="76083868-aa34-47dd-834b-d4b0bc965410" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-HTTP:NOT_FOUND" doc:id="c5cf0e7c-f59c-4ddd-9453-9685912f7312" expectedErrorType="HTTP:NOT_FOUND" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="48b955ab-e1ea-476f-96c3-6177a6c06e9d" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="HTTP:NOT_FOUND" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="f31847af-4475-444e-b5bb-ee81595d017c" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="17c6bc61-5825-4178-be0f-b461a5734d58" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="2734904d-8b08-4c52-81b8-d43afbb3b6b8" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-HTTP:UNAUTHORIZED" doc:id="e65149d8-8b42-4ac2-99cf-053a543027a7" expectedErrorType="HTTP:UNAUTHORIZED" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="7da4c353-0f99-4e96-802c-de8e10633c69" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="HTTP:UNAUTHORIZED" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="f5b33348-22fa-48f0-a04c-a78bf58442b0" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="eca588fc-4753-4852-83bb-5831c03c85e7" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Copy_of_Flow-ref to doit-hubspot-sapi-main" doc:id="2b046c20-2fc0-4a90-a2bb-b4c6015f4527" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-HTTP:UNSUPPORTED_MEDIA_TYPE" doc:id="a0e16c38-aae8-45e5-a018-1579afdf2c52" expectedErrorType="HTTP:UNSUPPORTED_MEDIA_TYPE" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="a0c1bcc2-aac2-44d9-974c-7f8aa0d866a1" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="HTTP:UNSUPPORTED_MEDIA_TYPE" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="2b82afec-3ecf-403c-bc74-37fac79abf53" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="44872bcc-b334-48f2-8715-d665c3397d82" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="779ce6b4-9ec2-4dea-8ad5-b6a85dc7b1a6" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-STREAM_MAXIMUM_SIZE_EXCEEDED" doc:id="7dc304cc-28a2-4df6-9501-8c9e8a78809b" expectedErrorType="STREAM_MAXIMUM_SIZE_EXCEEDED" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="c068af7e-181e-4888-901b-0bf50c63e610" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="STREAM_MAXIMUM_SIZE_EXCEEDED" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="a2038634-e8e5-4f83-9aae-c91d3bc773ad" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="ce3df615-42e2-40fc-a52d-6fa184652b2c" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="e4ff6430-dce0-4114-b285-68136ae06ef1" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-RETRY_EXHAUSTED" doc:id="40714c3b-8bdc-4288-a96f-643533ea8171" expectedErrorType="RETRY_EXHAUSTED" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="e160424c-97de-4b57-b89e-afceaadb3b15" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="RETRY_EXHAUSTED" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="3cc3f426-25ad-4bca-8420-6ce192bc6300" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="f0891cc9-81a4-40bd-9dd3-0ad83756b096" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="4d2bf9ca-49b0-442a-98bd-ab31d7d5dbf9" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-REDELIVERY_EXHAUSTED" doc:id="9406ec76-9ce3-44bb-8298-a99c50df05f7" expectedErrorType="REDELIVERY_EXHAUSTED" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="15aca623-296c-4406-bc21-feb622fdb91d" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="REDELIVERY_EXHAUSTED" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="57809f3d-ed84-4e0a-803c-c266703ccc29" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="9fff4b9e-4111-4784-98b2-ab6fe8c00480" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="c8c6d9f2-5c18-4174-8e01-61b92ad7a9e0" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-CONNECTIVITY" doc:id="4fd5b8c0-5709-44bd-bd41-55b4ea8cf673" expectedErrorType="CONNECTIVITY" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="da4f1f19-4493-4f50-bbfa-6f2a4d4e803d" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="CONNECTIVITY" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="76cefad5-b158-4c7c-9308-ddb519fae003" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="ecd9ccf8-77da-4ece-942b-00b5fa0d9b87" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="606bcc50-6453-4f5e-8302-a4561b6428bc" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-TIMEOUT" doc:id="9cb76fd0-f367-4201-9145-ea181585c743" expectedErrorType="TIMEOUT" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="21c42714-95c8-48e0-a3c6-40c50d2de972" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="TIMEOUT" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="f1ba11f6-c5b0-4ecb-82a3-e9b6ab7a6731" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="1e6bb33d-8c5b-4b1a-bd81-3c3735d0a669" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Copy_of_Flow-ref to doit-hubspot-sapi-main" doc:id="e4a42a7d-de23-40b5-8139-2bfe82ce98a3" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-TRANSFORMATION" doc:id="141f2234-7aa9-4272-a93a-ddebfa89c68e" expectedErrorType="TRANSFORMATION" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="e8a8b74e-8772-41e6-b48b-9f48259d23f5" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="TRANSFORMATION" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="407a4a34-fc10-4c7c-9b81-874abbac15a2" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="47a59d30-7776-4cab-8c4a-fb4b59fbf772" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="fa95f4f9-3748-48ed-b40a-741c43cb23a3" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-ROUTING" doc:id="eaebb08c-e53a-4b71-ac3b-6bd28b919a56" expectedErrorType="ROUTING" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="f352fbfc-fd93-451c-bc1b-774f89e6a27f" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="ROUTING" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="ae76bc38-fb79-4010-9934-dbceca858ff5" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="6110587f-6738-4909-8231-d43a3c049678" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="4e9148a7-7a9f-4b9f-b311-f93000ce67e2" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>
	<munit:test name="global-error-suiteTest-SECURITY" doc:id="f4981cf6-7b1e-40f3-9d8a-7164f83ff2c1" expectedErrorType="SECURITY" >
		<munit:behavior >
			<munit-tools:mock-when doc:name="Mock apikit:router" doc:id="64f446cd-cbbe-45d2-9843-c8296a9a043f" processor="apikit:router" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="doit-hubspot-sapi-config" attributeName="config-ref" />
				</munit-tools:with-attributes>
				<munit-tools:then-return >
					<munit-tools:error typeId="SECURITY" />
				</munit-tools:then-return>
			</munit-tools:mock-when>
			<munit-tools:mock-when doc:name="Mock flow-ref to global-exception-response-sub-flow" doc:id="e69a4506-05bc-4566-b9c1-44007aee260b" processor="flow-ref" >
				<munit-tools:with-attributes >
					<munit-tools:with-attribute whereValue="Refer to global-exception-response-sub-flow" attributeName="doc:name" />
					<munit-tools:with-attribute whereValue="8700cad3-a435-45cf-b679-d08ecffdd3bf" attributeName="doc:id" />
				</munit-tools:with-attributes>

			</munit-tools:mock-when>
		</munit:behavior>
		<munit:execution >
			<flow-ref doc:name="Flow-ref to doit-hubspot-sapi-main" doc:id="5147f909-93cc-443f-892c-47106376cd94" name="doit-hubspot-sapi-main" />
		</munit:execution>
	</munit:test>

</mule>
